test/new-e2e/pkg/runner/ci_profile.go

```diff
@@ -41,15 +41,31 @@ func NewCIProfile() (Profile, error) {
 	// TODO move to job script
 	os.Setenv("PULUMI_CONFIG_PASSPHRASE", passVal)
 
+	// Test early create resources
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Leftover?